src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -1903,21 +1903,17 @@ public List<ModificationInfos> applyModifications(Network network, UUID networkU
     ** Build variant : sending message to rabbitmq
      */
     public void buildVariant(UUID networkUuid, BuildInfos buildInfos, String receiver) {
-        sendRunBuildMessage(new BuildExecContext(networkUuid, buildInfos, receiver).toMessage(objectMapper));
-    }
-
-    private void sendRunBuildMessage(Message<String> message) {
-        RUN_MESSAGE_LOGGER.debug("Sending message : {}", message);
-        publisher.send("publishBuild-out-0", message);
+        String buildInfosJson;
```

Comment:
Why do you remove `BuildExecContext::toMessage` ?

Reply:
BuildExecContext::toMessage build the message that will be sent in notification. That work is done directly in the NotificationService now.